src/stories/Library/material-grid-related/material-grid-related.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import { ReactComponent as TriangleIcon } from "../../../public/icons/basic/icon-triangle.svg";
+import { MaterialGrid } from "../material-grid/MaterialGrid";
+import MaterialGridData from "../material-grid/MaterialGridData";
+
+const MaterialGridRelated: React.FC = () => {
+  return (
+    <div className="material-grid-related">
+      <div className="material-grid-related__header">
+        <h2 className="material-grid-related__title">Andre materialer</h2>
+        {/* Mobile display : Dropdown */}
+        <div className="dropdown dropdown--grey-borders input-with-dropdown__dropdown material-grid-related__dropdown">
+          <select
+            className="dropdown__select dropdown__select--inline focus-styling"
+            aria-label="Select material filter"
+            value="recommendation"
+          >
+            <option value="recommendation">Recommendation</option>
+            <option value="series">Series</option>
+            <option value="author">Author</option>
+          </select>
+          <div className="dropdown__arrows dropdown__arrows--inline">
+            <TriangleIcon
+              className="dropdown__arrow dropdown__arrow--bottom"
+              aria-hidden="true"
+            />
+          </div>
+        </div>
```

Comment:
Please consider if this could be reduced to use the existing `Dropdown` component.